packages/react/slider/src/Slider.tsx

```diff
@@ -465,19 +376,29 @@ const SliderPart = React.forwardRef((props, forwardedRef) => {
           event.preventDefault();
         }
       })}
-      /**
-       * Prevent pointer events on other elements on the page while sliding.
-       * For example, stops hover states from triggering on buttons if
-       * mouse moves over a button during slide.
-       *
-       * Also ensures that slider receives all pointer events after mouse down
-       * even when mouse moves outside the document.
-       */
       onPointerDown={composeEventHandlers(props.onPointerDown, (event) => {
-        (event.target as HTMLElement).setPointerCapture(event.pointerId);
+        const target = event.target as HTMLElement;
+        target.setPointerCapture(event.pointerId);
+        // Prevent browser focus behaviour because we focus a thumb manually when values change.
+        event.preventDefault();
```

Comment:
This change allowed me to remove the `setTimeout` stuff from [here](https://github.com/radix-ui/primitives/pull/731/files#diff-6bc9aae1ceceb7cfd3e1b1f419113c46a6d88dd4515e280bac28c9b1313eae99L101). We couldn't do this before because of [this](https://github.com/radix-ui/primitives/pull/731/files#diff-6bc9aae1ceceb7cfd3e1b1f419113c46a6d88dd4515e280bac28c9b1313eae99L440) but we're no longer using mouse events.